src/render/blog/release-2_0_0.html.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Knot.x 2.0 released!
+description: We have just released a major release Knot.x 2.0.
+author: skejven
+keywords: release
+order: 1
+date: 2019-08-23
+knotxVersions:
+    - 2.0.0
+---
+# Knot.x 2.0
+Hurray,
+
+Knot.x 2.0 - the open-source integrations framework is finally here!
+It comes with the HTTP server, distribution and platform for building backend APIs.
+The main changes and features of the second version are:
+- `Design First approach`: We get rid of custom routing definitions in favor of using standards. The [Open API v3](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md) is the way routing is defined in Knot.x server. That means you can start designing API before any
+line of code is written and use it as the documentation for the developers and clients.
+- `Backend API`: In the previous version, building API using Knot.x could be cumbersome in some cases. With `2.0` we bring completely new experience for building any kind of backend API
```

Comment:
This section can be combined with the previous one. I can see three major topics in this area:

- design first approach
- configurable integrations
- stability patterns

Reply:
With `2.0` we bring completely new experience for building any kind of backend API such as:
- Gateway API: a logic combining data from different sources, introducing caches or fallbacks
- Backend For Frontend: meets SPA challenges and allows to introduce RESTful or GraphQL contracts between your frontend and backend applications
- HTTP / Web API: exposes any kind of datasource with HTTP-based API